ui/react-components/components/AppointmentEditor/AppointmentEditorService.js

```diff
@@ -12,3 +13,8 @@ export const saveAppointment = async (appointment) => {
     getFormattedProviders(appointment.providers);
```

Comment:
Use forEach if the intention is not to use return array

Reply:
Changed